python-rhsm/src/rhsm/connection.py

```diff
@@ -1405,6 +1405,20 @@ def getSubscriptionList(self, owner_key):
         results = self.conn.request_get(method)
         return results
 
+    def updateSubscriptionList(self, owner_key, auto_create_owner=False, lazy_regen=True):
+        """
+        Update subscriptions for a particular owner.
+        """
+        method = "/owners/%s/subscriptions" % self.sanitize(owner_key)
+
+        if auto_create_owner:
+            method += "&auto_create_owner=true"
+        if lazy_regen:
```

Comment:
These should be done a little differently. As written if you pass `lazy_regen=False`, then the query param is not added at all, and thus candlepin uses default value which is `true`, which is a little surprising. I would suggest we do something like this instead:

```python
if auto_create_owner is not None:
    method += "&auto_create_owner=%s" % bool(auto_create_owner).lower()
if lazy_regen is not None:
    method += "&lazy_regen=%s" % bool(lazy_regen).lower()
```